quarkus-integration/quarkus/deployment/src/main/java/ai/timefold/solver/quarkus/deployment/TimefoldProcessor.java

```diff
@@ -210,12 +209,13 @@ SolverConfigBuildItem recordAndRegisterBuildTimeBeans(CombinedIndexBuildItem com
         // Only skip this extension if everything is missing. Otherwise, if some parts are missing, fail fast later.
         if (indexView.getAnnotations(DotNames.PLANNING_SOLUTION).isEmpty()
                 && indexView.getAnnotations(DotNames.PLANNING_ENTITY).isEmpty()) {
-            log.warn("Skipping Timefold extension because there are no @" + PlanningSolution.class.getSimpleName()
-                    + " or @" + PlanningEntity.class.getSimpleName() + " annotated classes."
-                    + "\nIf your domain classes are located in a dependency of this project, maybe try generating"
-                    + " the Jandex index by using the jandex-maven-plugin in that dependency, or by adding"
-                    + "application.properties entries (quarkus.index-dependency.<name>.group-id"
-                    + " and quarkus.index-dependency.<name>.artifact-id).");
+            log.warn(
+                    """
+                            Skipping Timefold extension because there are no @%s or @%s annotated classes.
+                            If your domain classes are located in a dependency of this project, maybe try generating the \
+                            Jandex index by using the jandex-maven-plugin in that dependency, or by addingapplication.properties entries \
+                            (quarkus.index-dependency.<name>.group-id and quarkus.index-dependency.<name>.artifact-id)."""
+                            .formatted(PlanningSolution.class.getSimpleName(), PlanningEntity.class.getSimpleName()));
```

Comment:
This is a logging message. Instead of using our own formatting, use the logger's native way of passing parameters to log messages.

Reply:
We are using `org.jboss.logging.Logger`, but its API does not provide an easy way to pass parameters.